src/heads-up/components/properties-item.vue

```diff
@@ -147,14 +152,20 @@ export default {
       this.info = this.schema[this.keyName].meta.info;
       this.link = this.schema[this.keyName].meta.link;
     }
-
-    this.errors = validateSchema({
-      schema: this.schema,
-      key: this.keyName,
-      value: this.value,
-      attrs: this.attrs,
-    })
-  }
+    this.validateSchema()
+  },
+  methods:{
+    validateSchema(){
+      const schemaTemplate = {
+        schema: this.schema,
+        key: this.keyName,
+        value: this.value,
+        attrs: this.attrs
+      }
+      
+      this.errors = validateSchema(schemaTemplate)
```

Comment:
I put this schemaTemplate in a const to reuse later on. I am planning to create multiple validation functions like rule 166